.gitattributes

```diff
@@ -0,0 +1 @@
+.git_patch export-subst
```

Comment:
this file need to be ignore from manifest and different tools in the same way `.gitignore` is ignored lol
need to update
```
Porting/Maintainers.pl
Porting/manicheck
installperl
```

`t/porting/manifest.t` is already patched

Porting/makerel seems to already cover the `.git_patch` with `$_ !~ m!^\.(?:git|github)`

Reply:
I've updated `Porting/manicheck`.  The exclusions in `installperl` only seem relevant for the `lib` directory, and `Porting/Maintainers.pl` only seems to be about the dist directories in `ext`/`dist`/`cpan`.